docs/articles/bankid.md

```diff
@@ -968,14 +969,111 @@ services.RemoveAll(typeof(IBankIdResultStore));
 
 ### Resolve the end user ip
 
-In some scenarios, like running behind a proxy, you might want to resolve the end user IP yourself and override the default implementaion.
+In some scenarios, like running behind a proxy, you might want to resolve the end user IP yourself and override the default implementation.
 
 Either register a class implementing `IBankIdEndUserIpResolver`:
 
 ```csharp
 services.AddTransient<IBankIdEndUserIpResolver, EndUserIpResolver>();
 ```
 
+---
+### Resolve the end user device data
+
+The User Device feature allows you to include detailed information about the end user's device in BankID authentication and signing requests. This feature improves risk evaluation and aligns with BankID's recommended implementation for risk indication.
+
+When using BankID, device information provides valuable metadata that enhances security and ensures a smoother user experience. Including the **device type** (e.g., `APP` or `WEB`) helps BankID:
+- **Evaluate risk** for each request.
+- **Take automated actions** based on high-risk scenarios (if enabled).
+- Provide **better insights** into the context of the request.
+
+#### Configuring User Device
+
+To customize the User Device feature, use the `UseDeviceData` extension in the BankID client builder. This allows you to specify the device type and any relevant metadata using resolvers.
+
+Active Login has a default implementation of the User Device feature that uses a web browser as the device type. You can customize the default implementation or create your own.
+
+The following implementation is necessary to use the User Device feature:
+- IBankIdEndUserDeviceDataResolverFactory: A factory that provides the resolvers for the device type.
+- IBankIdEndUserDeviceDataResolver: A resolver that provides the device data for the device type.
+- IBankIdEndUserDeviceDataConfiguration: A configuration that specifies the device type to use.
+
+The BankIdFlowService will automatically include the device data in the BankID request.
+
+##### Configuration examples with the default implementation
+
+*For requests initiated by av web browser:*
```

Comment:
Adjust example to use a custom implementation of the interfaces required for end user device data. To make it more clear that no configuration is required for using the default implementation.

Reply:
I have updated the documentation and also added more information (made it clearer imho).